scene/resources/curve.cpp

```diff
@@ -479,6 +479,9 @@ void Curve::set_bake_resolution(int p_resolution) {
 }
 
 real_t Curve::sample_baked(real_t p_offset) const {
+	// Make sure that p_offset is finite
```

Comment:
```suggestion
	// Make sure that p_offset is finite.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide). For all